### ISSUE TITLE
Possible Bug in Hubert Fine tuning in script : https://github.com/pytorch/audio/blob/main/examples/hubert/lightning_modules.py

### ISSUE THREAD

**Issue by varun-krishnaps** [pytorch/audio#3452]
### 🐛 Describe the bug

1. In the script : https://github.com/pytorch/audio/blob/main/examples/hubert/lightning_modules.py ; There seems to be issue in the line number 431 and 438 : **x, _ = self.model.mask_generator(x, padding_mask)** ;
 I Don't think we need to mask the feature extractor outputs during fine tuning stage , it's only required while Pre-training the model.

2. It isn't clear how to run the finetune.py script, Does the data preparation require any tsv preparation like the pretraining stage ?? Or we just give the dataset path ??

### Versions

It doesn't depend on the version

**Comment by nateanl** [comment]
masking is still required during finet uning stage, see https://github.com/facebookresearch/fairseq/blob/main/examples/hubert/config/finetune/base_10h.yaml#L65

To run the finetune.py script, please check https://github.com/pytorch/audio/tree/main/examples/hubert#fine-tuning-usage
